nexus-ingest/nexus-sink/src/main/java/org/nasa/jpl/nexus/ingest/nexussink/S3Store.java

```diff
@@ -0,0 +1,65 @@
+package org.nasa.jpl.nexus.ingest.nexussink;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.*;
+import org.nasa.jpl.nexus.ingest.wiretypes.NexusContent.NexusTile;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.Collection;
+
+/**
+ * Created by djsilvan on 6/26/17.
+ */
+public class S3Store implements DataStore {
+
+    private AmazonS3 s3;
+    private String bucketName;
+
+    public S3Store(AmazonS3Client s3client, String bucketName) {
+        s3 = s3client;
+        this.bucketName = bucketName;
+    }
+
+    public void saveData(Collection<NexusTile> nexusTiles) {
+
+        for (NexusTile tile : nexusTiles) {
+            String tileId = getTileId(tile);
+            byte[] tileData = getTileData(tile);
+            Long contentLength = (long) tileData.length;
+            InputStream stream = new ByteArrayInputStream(tileData);
+            ObjectMetadata meta = new ObjectMetadata();
+            meta.setContentLength(contentLength);
+
+            try {
+                s3.putObject(new PutObjectRequest(bucketName, tileId, stream, meta));
+            }
+            catch (AmazonServiceException ase) {
```

Comment:
Error handling needs to be improved. At a minimum make it proper logging calls not `System.out.println`.

Should raise exceptions as mentioned in previous comment on `DynamoStore`